src/EnergyPlus/CondenserLoopTowers.cc

```diff
@@ -6355,6 +6344,7 @@ namespace CondenserLoopTowers {
         // This subroutine is for passing results to the outlet water node.
 
         // set node information
+        PlantUtilities::SafeCopyPlantNode(state, this->WaterInletNodeNum, this->WaterOutletNodeNum);
```

Comment:
@RKStrand hope you don't mind me jumping in here.  When components are in series, this is needed to pass data down the branch to the next component.

Reply:
Not at all, @EnergyArchmage!  Thanks for making that change.